### DISCUSSION TITLE
Single withdrawal, multiple expense accounts?

### DISCUSSION THREAD

**Discussion by Ezwen** [firefly-iii/firefly-iii#7407] [Firefly III] [answered]
Is it possible with Firefly, for a single withdrawal (eg. 100€ going out of a bank account / asset) to split the amount towards two different expense accounts? (eg. 25€ going to expense account A and 75€ going to expense account B)

Same question for splitting a single deposit into multiple revenue accounts.

Thanks in advance!

**Comment by JC5** [comment, marked as answer]
Yes to both.

**Reply by Ezwen** [reply]
great, thanks!
